### PR TITLE
Migrate tests to jest

### PR DESCRIPTION
**Pull request by ldeavila**
Migrated all wildcat tests to jest

### CODE REVIEW

File: .jestrc.memory.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "collectCoverage": true,
+    "collectCoverageFrom": ["/packages"],
+    "coverageDirectory": "coverage/memory",
```
Comment (jamsea) [inline .jestrc.memory.json]: Why do we need a separate .jestrc file for memory?

File: example/package.json [hunk 2]
```diff
@@ -58,7 +58,7 @@
         "file-loader": "^0.10.0",
         "flow-runtime": "^0.3.0",
         "fs-extra": "^2.0.0",
-        "happypack": "^3.0.2",
+        "happypack": "3.1.0",
```
Comment (doctyper) [inline example/package.json]: `^3.1.0`

File: package.json [hunk 3]
```diff
@@ -74,11 +77,13 @@
         "shelljs": "^0.7.0",
         "sinon": "^2.0.0-pre.5",
         "sinon-chai": "^2.8.0",
-        "webpack": "^2.2.1",
+        "webpack": "2.6.1",
```
Comment (doctyper) [inline package.json]: `^2.6.1`

File: packages/react-wildcat/test/utils/webpackBundleValidationSpec.js [hunk 4]
```diff
@@ -4,207 +4,156 @@ const sinon = require("sinon");
 const sinonChai = require("sinon-chai");
```
Comment (doctyper) [inline packages/react-wildcat/test/utils/webpackBundleValidationSpec.js]: Something funky happened here? You replaced all of the tests with unrelated tests.

File: example/package.json (hunk 2 above)
Comment (ldeavila) [inline example/package.json]: 3eed25b

File: package.json (hunk 3 above)
Comment (ldeavila) [inline package.json]: 3eed25b

**Pull request merged**